### PR TITLE
Elastic Stack Kafka Monitoring

### PR DESCRIPTION
**Pull request by urso**
Sample files for Kafka Monitoring with Beats blog series.

### CODE REVIEW

**Comment by andrewkroh** [comment]
LGTM.

@asawariS Are there any naming conventions to follow for the top-level directory names in this repo?

File: Blog_ElasicStack_Kafka/1_filebeat/Readme.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    $ filebeat -e -v -c filebeat.yml -E filebeat.registry_file=kafka1.registry -E output.elasticsearch.hosts='localhost:9200' -E name=kafka1 -E kafka.home="$(pwd)/logs/kafka1"
+    ...
+
+    $ filebeat -e -v -c filebeat.yml -E filebeat.registry_file=kafka2.registry -E output.elasticsearch.hosts='localhost:9200' -E name=kafka2 -E kafka.home="$(pwd)/logs/kafka2"
+    ...
+    
+6. Install Kibana dashboards
+
+    $ import_dashbaords -es http://localhost:9200 -user elastic -pass changeme -dir kibana
```
Comment (gingerwizard) [inline Blog_ElasicStack_Kafka/1_filebeat/Readme.md]: should this be import_dashboards?

Comment (urso) [inline Blog_ElasicStack_Kafka/1_filebeat/Readme.md]: yes. `import_dashbaords` is a tool shipped with beats: `filebeat-5.1.1-darwin-x86_64/scripts/import_dashboards`. This tool imports dashbaords exported beats dev tools  right into kafka index. It's common for beats related dashboards to use it.

**Comment by andrewkroh** [comment]
Merged b/c the blog post is live now.

**Pull request merged**